README.md

```diff
@@ -20,7 +22,28 @@ Docker builds applications using a file called Dockerfile. Within the Dockerfile
 User Guide
 ==========
 ### What it does
-The tool takes in the following parameters: image name, tag, a potential store that holds images to tags. It will search GitHub for any repositories that have the specified image as a base image and replace them with the tag. The repositories are **NOT** automatically modified; pull requests are opened in that repository.
+The tool has three modes
+ 1. `all` - Reads store that declares the docker images and versions that you intend others to use. 
+ Example:
+```
+export git_api_url=https://api.github.com
+export git_api_token=my_github_token
+docker run --rm -e git_api_token -e git_api_url salesforce/dockerfile-image-update all image-to-tag-store
+```
+ 2. `parent` - Searches github for images that use a specified image name and sends pull requests if the image tag doesn't match intended tag. The intended image with tag is passed in the command line parameters. The intended image-to-tag mapping is persisted in a store in a specified git repository under the token owner. 
+Example:
+```
+export git_api_url=https://api.github.com
+export git_api_token=my_github_token
+docker run --rm -e git_api_token -e git_api_url salesforce/dockerfile-image-update parent my_org/my_image v1.0.1 image-to-tag-store
+```
+ 3. `child` -  
+The tool takes in the following parameters: image name, tag, and git repository for persistent storage of . It will search GitHub for any repositories that have the specified image as a base image and replace them with the tag. The repositories are **NOT** automatically modified; pull requests are opened in that repository.
```

Comment:
`persistent storage of `...? I don't think you finished your thought there.

Reply:
Ooops, good catch. Fixed!